tasks/deployMerkleDistributor.js

```diff
@@ -0,0 +1,23 @@
+require('@nomiclabs/hardhat-ethers')
```

Comment:
Use ES Modules syntax:
```suggestion
import "@nomiclabs/hardhat-ethers";
```